lib/tasks/npm-task.js

```diff
@@ -25,8 +25,6 @@ class NpmTask extends Task {
 
     // The command to run: can be 'install' or 'uninstall'
     this.command = '';
-
-    this.versionConstraints = '3 || 4 || 5 || 6 || 7 || 8';
```

Comment:
Seems better to not check this anymore than to keep bumping this every new major.

Reply:
I imagine `engines` in package.json should be sufficient, too

Reply:
I think so yes. We could even add `"npm": ">= 3"`, if we want to keep erroring on NPM versions below v3 I think? Though, not sure if that's really needed.